README.md

```diff
@@ -13,10 +13,10 @@ instead of a container runtime.
 ## Documentation
 
 If you're new to the project, get started with [the
-introduction](docs/intro/README.md). For more in-depth information about
-Krustlet, plunge right into the [topic guides](docs/topics/README.md).
+introduction](https://docs.krustlet.dev/intro). For more in-depth information about
```

Comment:
links no worky; I think we need to update the redirect setup on netlify for docs.krustlet.dev

Reply:
links work!